### PR TITLE
fix: adjusted project status message based on search state gf-313

### PR DESCRIPTION
**Pull request by Fjortis**
Modified the logic for displaying the project status message to handle two cases:
  - When no projects have been created: "No projects created yet. Create the first project now."
  - When a search query returns no results: "No projects found matching your search criteria. Please try different keywords."
  
 No Projects
No message
No Projects
No projects
With projects
with proj
With Projects
with proj no search result

### CODE REVIEW

File: apps/frontend/src/pages/projects/projects.tsx [hunk 1]
```diff
@@ -68,6 +68,10 @@ const Projects = (): JSX.Element => {
 	);
 
 	const hasProject = projects.length !== EMPTY_LENGTH;
+	const hasSearch = search.length !== EMPTY_LENGTH;
+	const ProjectStatusMessage = hasSearch
```
Comment (liza-veis) [inline apps/frontend/src/pages/projects/projects.tsx:72]:
```suggestion
	const emptyPlaceholderMessage = hasSearch
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/projects/projects.tsx (hunk 1 above)
Comment (Fjortis) [reply apps/frontend/src/pages/projects/projects.tsx]: Adjusted the naming

**Review by liza-veis — APPROVED** [review]

File: apps/frontend/src/pages/projects/projects.tsx [hunk 2]
```diff
@@ -68,6 +68,10 @@ const Projects = (): JSX.Element => {
 	);
 
 	const hasProject = projects.length !== EMPTY_LENGTH;
+	const hasSearch = search.length !== EMPTY_LENGTH;
+	const emtyPlaceholderMessage = hasSearch
```
Comment (GvoFor) [inline apps/frontend/src/pages/projects/projects.tsx:72]:
```suggestion
	const emptyPlaceholderMessage = hasSearch
```

Comment (Fjortis) [reply apps/frontend/src/pages/projects/projects.tsx:72]: Nice catch buddy 👍

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**